### PR TITLE
[CDAP-8626][CDAP-8627] Calls to action for one click deploy

### PR DESCRIPTION
**Pull request by tonybach**
JIRA: https://issues.cask.co/browse/CDAP-8626

- Added CTA for one click deploy for apps
- Added CTA for one click deploy for plugins
- Updated entity list view after the above actions

### CODE REVIEW

**Comment by elfenheart** [comment]
@tonybach I think we should add the artifact upload event emitter to all the artifact wizards

**Comment by tonybach** [comment]
Changed the styling of the deploy wizard as spec'd out in this JIRA: https://issues.cask.co/browse/CDAP-8627

File: cdap-ui/app/cdap/components/CaskWizards/OneStepDeploy/Deploy/Deploy.scss [hunk 1]
```diff
@@ -17,4 +17,24 @@
...
+    }
+  }
+}
+
+.cask-wizard {
+  .wizard-body {
+    .wizard-steps-content {
+      .wizard-navigation {
+        .btn {
+          margin-top: 10px;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/CaskWizards/OneStepDeploy/Deploy/Deploy.scss]: I think this has caused an un-necessary overflow scroll to appear in other wizards.

**Review by ajainarayanan — COMMENTED** [review]
Just a minor comment. Othewise 👍 LGTM

**Comment by elfenheart** [comment]
👍  LGTM

**Comment by tonybach** [comment]
Build: http://builds.cask.co/browse/CDAP-DRC5868

**Pull request merged**